### DISCUSSION TITLE
Memory Config

### DISCUSSION THREAD

**Discussion by dmamira** [vercel/vercel#4382] [General] [answered]
I'm using next.js.
For this reason, the API is placed under /pages/api.
Then, I created the now.json file as follows to change the memory allocation of the serverless function.
`"functions":{
        "pages/api/bid.ts":{
            "memory":3008
        }
    }`
However, the memory allocation did not change.
Next.How do you change the memory allocation if you are using js?
Also, is it possible to change the memory allocation for pages containing getServerSideProps?

**Comment by dmamira** [comment]
and I'm using Pro Plan

**Comment by paulogdm** [comment, marked as answer]
Hello,

Yes, it is possible to change the memory allocation for any serverless function, including pages. Please drop a message to us on vercel.com/support.
